src/ui/components/Button/Button.tsx

```diff
@@ -4,11 +4,13 @@ import styled from 'styled-components/native'
 import { Text, TouchableOpacity } from 'react-native'
 import theme from 'ui/theme'
 
+const noop = () => {}
+
 interface ButtonProps {
-  size: 'default' | 'small'
   type: 'border' | 'fill' | 'transparent'
   label: string
```

Comment:
i like that you reordered the props to have the required ones first. i'm going to start doing this.